### PR TITLE
Removed some unused variables that were causing IDE warnings.

### PR DESCRIPTION
**Pull request by HoneySkull**
Removed unused variables causing warnings in the IDE (Eclipse).  This PR is associated with #2800

### CODE REVIEW

File: megamek/src/megamek/client/ui/swing/CommonMenuBar.java [hunk 1]
```diff
@@ -1085,7 +1084,7 @@ public synchronized void setBoard(boolean available) {
      *            cleared.
      */
     public synchronized void setUnitList(boolean available) {
-        hasUnitList = available;
+        //manageMenu sets unit menus based on phase not from this setUnitListFlag.
```
Comment (Windchild292) [inline megamek/src/megamek/client/ui/swing/CommonMenuBar.java:1087]:
```suggestion
        // manageMenu sets unit menus based on phase not from this setUnitListFlag.
```

File: megamek/unittests/megamek/client/bot/princess/NewtonAerospacePathRankerTest.java [hunk 2]
```diff
@@ -29,10 +27,6 @@
 @RunWith(JUnit4.class)
 public class NewtonAerospacePathRankerTest {
 
-    private Princess mockPrincess;
-    private NewtonianAerospacePathRanker mockPathRanker;
-    private IGame mockGame;
-    private GameOptions mockGameOptions;
```
Comment (Windchild292) [inline megamek/unittests/megamek/client/bot/princess/NewtonAerospacePathRankerTest.java:35]: Given how the below is... we should leave these here and ignore all of the tests.

**Review by Windchild292 — COMMENTED** [review]
Few comments first

File: megamek/unittests/megamek/client/bot/princess/NewtonAerospacePathRankerTest.java (hunk 2 above)
Comment (HoneySkull) [reply megamek/unittests/megamek/client/bot/princess/NewtonAerospacePathRankerTest.java]: Added @SupressWarnings to leave the variables in tact.  The commented out test has the @ignore JUnit annotation.

**Review by Windchild292 — APPROVED** [review]

**Pull request merged**